newswires/client/src/App.tsx

```diff
@@ -188,6 +249,7 @@ export function App() {
 						  }
 						`}
 						height: 100%;
+						max-height: 100vh;
```

Comment:
this seems to be needed for the vertical split, although I couldn't quite work out why this wasn't handled by the restrictions already put on the parent element..